services/scribe/db/receipt_test.go

```diff
@@ -186,3 +187,33 @@ func (t *DBSuite) MakeRandomReceipt(txHash common.Hash) types.Receipt {
 		TransactionIndex:  uint(gofakeit.Uint64()),
 	}
 }
+
+func (t *DBSuite) TestRetrieveReceiptsWithStaleBlockHash() {
+	t.RunOnAllDBs(func(testDB db.EventDB) {
+		chainID := gofakeit.Uint32()
+		startBlock := 2
+		hash1 := common.BigToHash(big.NewInt(1))
+		hash2 := common.BigToHash(big.NewInt(2))
+		hash3 := common.BigToHash(big.NewInt(3))
+		newHash2 := common.BigToHash(big.NewInt(22))
+		blockHashes := []common.Hash{hash1, hash2, hash3}
+		blockHashesStr := []string{hash1.String(), newHash2.String(), hash3.String()}
+
+		for i := 0; i < 10; i++ {
+			receipt := t.MakeRandomReceipt(common.BigToHash(big.NewInt(gofakeit.Int64())))
+			receipt.BlockNumber = big.NewInt(int64(i))
+			fmt.Println("SSS", i, blockHashes[i%3])
```

Comment:
??

Reply:
is this questioning the print or something else

Reply:
is this questioning the printout or something else